solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -61,21 +61,30 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         uint128 removeAverageRateD; // average rate of 1 A in units of B when liquidity is removed (denominator)
     }
 
+    struct PoolIndex {
+        bool isValid;
+        uint256 value;
+    }
+
     IERC20Token internal constant ETH_RESERVE_ADDRESS = IERC20Token(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
     uint32 internal constant PPM_RESOLUTION = 1000000;
     uint256 internal constant MAX_UINT128 = 2**128 - 1;
 
     // the address of the whitelist administrator
     address public whitelistAdmin;
 
+    // list of pools with less minting restrictions and mapping of pool anchor address -> index in the list of pools for quick access
+    IConverterAnchor[] private _highTierPools;
+    mapping(IConverterAnchor => PoolIndex) private highTierPoolIndices;
```

Comment:
Should this state variable also start with an underscore (like `_highTierPools`)?

Reply:
I just did it cause we have such a function name

Reply:
Gotcha